src/en/help-vmware.md

```diff
@@ -70,18 +70,48 @@ juju add-credential myvscloud
 ```
 The process now becomes interactive. You will first be asked for an arbitrary
 name for this credential, which you choose for yourself, followed by the
-username and password for your VMware installation. 
+username and password for your VMware installation.
+
+## Bootstrapping
 
 With credentials added, you can now start using Juju with your vSphere cloud:
 
 ```bash
 juju bootstrap myvscloud myvscontroller
 ```
 
-!!! Note: 
+There are three VMware-specific options you can use to specify the network and
+datatore to use.
+
+- **`primary-network`**: The primary network that VMs will be connected to. If
+  this is not specified, the first accessible network will be used.
+- **`external-network`**: The name of an additional "external" network to which
+  the VM should be connected. The IP from this network will be used as the
+  `public-address` of the VM's.
+- **`datastore`**: Datastore is the name of the datastore in which to create
+  the VM. If this is not specified, the first accessible datastore will be used.
+
+You can specify these options during bootstrap:
+
+```bash
+juju bootstrap myvscloud myvscontroller --config primary-network=PRIMARY_NET --config external-network=EXTERNAL_NET --config datastore=NFSSTORE
+```
+
+!!! Note:
+    When you specify these options in the bootstrap command, they will only
+    apply to the `controller` and `default` model. Use
+    [`model-defaults`](https://jujucharms.com/docs/2.2/models-config) if you
```

Comment:
Could you update this link to refer to `models-config` rather than the full URL (to avoid the specific version link)?